crates/model/src/solver_competition.rs

```diff
@@ -54,6 +54,7 @@ pub struct SolverSettlement {
     #[serde_as(as = "BTreeMap<_, HexOrDecimalU256>")]
     pub clearing_prices: BTreeMap<H160, U256>,
     pub orders: Vec<Order>,
+    pub is_winner: bool,
```

Comment:
I would add `#[serde(default)]` above the new field to make it explicit that `is_winner` might be missing for some entries and to make the deserialization more robust.

Reply:
Why not just do the migration in this PR as well to have al the data correct in one go?